eclipse-che/che-server
# build: Transition from Java EE to Jakarta EE

### What does this PR do?
The transition from Java EE to Jakarta EE
PR can be reviewed commit by commit.
TODO 
Everrest CQ - https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23669

### Screenshot/screencast of this PR
<img width="1670" alt="Знімок екрана 2021-08-10 о 16 28 12" src="https://user-images.githubusercontent.com/1614429/128875755-2468fb75-6df9-4f7e-a1f9-af6c5b2ad37e.png">
<img width="1680" alt="Знімок екрана 2021-08-10 о 16 27 36" src="https://user-images.githubusercontent.com/1614429/128875769-e94eaa46-1402-4f5b-af1e-a515670fcf3e.png">



### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/20247


### How to test this PR?
-  build or deploy che-server's image `quay.io/skabashn/che-server:jakarta_2021_09_03_14_12_20`
- login as "kubeadmin" on OpenShift
- Start new workspace.
- Workspace should start successfully.


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
